### DISCUSSION TITLE
How to delete old camera and events?

### DISCUSSION THREAD

**Discussion by arpitgupta** [blakeblackshear/frigate#3573] [Ask A Question] [answered]
Hi All,

Wondering what is the best way to delete a camera and events associated with it. Do i just delete the camera from the frigate configs and restart and it will clean up all recordings from that camera? Is there any clean up I need to do in HomeAssistant? 

I will be removing all camera's currently in frigate and replacing them so all existing events can be cleaned. I know where the data is stored but was not sure if I can just delete the data from the disk or is there any data in HA that would need to be cleaned up too? I was thinking I would just delete the Frigate integration and add it back when I had the new camera's setup.

**Comment by NickM-27** [comment, marked as answer]
Yes, remove the camera from the config and on the next cleanup frigate will remove the recordings and snapshots

**Reply by arpitgupta** [reply]
Thanks

I assume this is driven by the name of the camera in the config? So i wanted to add new camera's then I should wait until data gets cleaned up?

Is cleaning up the data manually in the media dir ok as well? Or there is information stored in HA that needs to be cleaned up as well?

**Reply by NickM-27** [reply]
It is based on name, if you let frigate run for 1 hour then any deleted cameras will be cleared out. 

If you did clear out manually it would also be fine.

**Reply by arpitgupta** [reply]
Thanks 👍

**Reply by yuckypants** [reply]
Is the only option to trigger cleanup setting the expire_interval? 

Also, do the events in the GUI directly correlate to a file? If so, if I delete the file using the web GUI, will it remove the file from storage? What about when I delete the file from storage, will it remove the file from the GUI?
